packages/react-table/src/components/TableComposable/examples/ComposableTable.md

```diff
@@ -311,6 +311,16 @@ To make a row draggable:
 ```ts isBeta file="ComposableTableDraggable.tsx"
 ```
 
+## Sticky table modifiers
+
+**Note:** Sticky table headers and columns have a higher `z-index` than the `z-index` used for menus (dropdown, select, etc). The intent is that the contents of a scrollable table will scroll under the sticky header/column, including any expanded menus. However, there may be use cases where a menu needs to appear on top of a sticky header/column, such as an expanded menu in a toolbar above a table with a sticky header.
+
+There are a few ways this can be handled:
+
+- Manipulate the `z-index` of the menu and/or table headers/columns manually.
+- Use the `menuAppendTo` prop in non-composable react components with menus to append the menu to an element outside of the table (e.g., the `<body>` element) so that the menu has a higher stacking context than - and can appear on top of - sticky headers/columns as well as appear outside of any scrollable content in the table.
```

Comment:
This may no longer be the recommendation after all the work @thatblindgeye has been doing on appending to `document.body`. We should double check what is the best ancestor to append a menu to to solve this problem.

Reply:
So I *believe* utilizing the `"parent"` value for the `menuAppendTo` prop would work as intended here?

This is only a thrown together attempt, but I added dropdowns to one of the Sticky examples in this PR's build below. Depending if that shows expected behavior, then it could make sense to either referring to "(e.g., the table's `parent` element)", or just omitting an example in this paragraph (other components should most likely have mention of the whole body vs parent thing).

https://user-images.githubusercontent.com/70952936/159538532-777686a9-844b-465a-b6c4-1bbccb6fa3e7.mov

Reply:
I'll update to reference the table's parent rather than the body for now at least.